examples/howto/plot_batch_simulate.py

```diff
@@ -113,18 +113,37 @@ def summary_func(results):
 # Extract and Plot Results
 ###############################################################################
 
+# Extract and overlay dipole waveforms
+dpl_waveforms = []
+for data_list in simulation_results['simulated_data']:
+    for data in data_list:
+        dpl_waveforms.append(data['dpl'][0].data['agg'])
+
+plt.figure(figsize=(10, 6))
+for waveform in dpl_waveforms:
+    plt.plot(waveform, alpha=0.5)
```

Comment:
```suggestion
    plt.plot(waveform, alpha=0.5, linewidth=2)
```
Try and see if thicker lines look a little better